tests/e2e/framework/kubernetes.go

```diff
@@ -712,6 +712,11 @@ func (k *KubeInfo) deployIstio() error {
 		log.Errorf("Unable to create namespace %s: %s", k.Namespace, err.Error())
 		return err
 	}
+	// Create istio-config namespace
+	if err := util.CreateNamespace("istio-config", k.KubeConfig); err != nil {
```

Comment:
this may be why this works in the gate, but doesn't work with @rshriram 's environment.  He may be running tiller locally which isn't executing this command.  I think a manual test would be in order.

Reply:
For circleci tests I noticed that the Istio-system namespace was getting created here. Hence adding the line for creating the Istio-config namespace here. After that change all circleci tests passed. At the moment a lot of tests including some circleci tests are failing so I am not sure what is what.